bionic/flow.py

```diff
@@ -959,6 +959,23 @@ def get(self, name, collection=None, fmt=None, mode=object):
         else:
             raise ValueError("Unrecognized collection type %r" % collection)
 
+    def get_docstring_for_entity(self, name):
+        """
+        Returns the docstring for function underlying entity ``name``.  Only supported for
+        FunctionProviders (not ValueProviders).  If no docstring is present, returns ``None``.
```

Comment:
`FunctionProvider` and `ValueProvider` are internal classes which the user probably won't be familiar with -- we should phrase this in the same language the documentation uses.  (E.g., "entities defined with functions".)  However, probably we could just say "if one is defined", since function-provided entities may also be missing docstrings, and presumably we will add the ability to put docstrings on value-provided entities later.